pkg/apis/vpc/v1alpha1/vpcnetworkconfiguration_types.go

```diff
@@ -9,9 +9,10 @@ import (
 )
 
 const (
-	AccessModePublic  string = "Public"
-	AccessModePrivate string = "Private"
-	AccessModeProject string = "PrivateTGW"
+	AccessModePublic       string = "Public"
+	AccessModePrivate      string = "Private"
+	AccessModeProject      string = "PrivateTGW"
+	AccessModeProjectInNSX string = "Private_TGW"
```

Comment:
you do not need define access mode here, you need to define them in subnet_type.go